Add ASA Stats on Metrics

salvatorecorvaglia opened the pull request:
# Description

_Please include a summary of the resource you're suggesting below:_

ASA Stats aims to have accurate and up-to-date price information for every ASA and NFT that has ever been sold on Algorand.


# Checklist

_Please make sure you reviewed the checklist and comply with each requirement:_

-   [x] My code follows the [contribution guidelines](../CONTRIBUTING.md) of this project
-   [x] This pull request has a title in the format `Add Name of Resource`:
    -   ✅ Add `py-algorand-sdk`
    -   ❌ Update readme.md

Commit:
Add ASA Stats on Metrics

ASA Stats aims to have accurate and up-to-date price information for every ASA and NFT that has ever been sold on Algorand.

aorumbayev reviewed (changes requested):
@salvatorecorvaglia  one more thing -> https://github.com/aorumbayev/awesome-algorand/runs/5201739670?check_suite_focus=true 

Add `.` at the end of the description sentence.

salvatorecorvaglia commented:
The point at the end has already been added.

aorumbayev commented:
@salvatorecorvaglia I've just re run the lint task still failing. Dot is not there yet, maybe you forgot to push to remote. Make sure to address the `.` before re-requesting the review. Should be an easy quick tweak from your side.

salvatorecorvaglia commented:
@aorumbayev i have this error:

Run npx awesome-lint
npm WARN exec The following package was not found and will be installed: awesome-lint
- Linting
✖ Linting

  README.md:252:[4](https://github.com/aorumbayev/awesome-algorand/runs/5201815300?check_suite_focus=true#step:4:4)1
  ✖  2[5](https://github.com/aorumbayev/awesome-algorand/runs/5201815300?check_suite_focus=true#step:4:5)2:41  List item description must end with proper punctuation  remark-lint:awesome-list-item

  1 error

Error: Process completed with exit code 1.

But the description is right:

Description
Please include a summary of the resource you're suggesting below:

ASA Stats aims to have accurate and up-to-date price information for every ASA and NFT that has ever been sold on Algorand.

Checklist
Please make sure you reviewed the checklist and comply with each requirement:

 My code follows the [contribution guidelines](https://github.com/aorumbayev/awesome-algorand/CONTRIBUTING.md) of this project
 This pull request has a title in the format Add Name of Resource:
✅ Add py-algorand-sdk
❌ Update readme.md

aorumbayev commented on README.md:
```diff
@@ -249,6 +249,7 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 - [Algorand Mempool](https://www.iamnotabot.com/pool) - Algorand mempool explorer.
 - [Metrika](https://app.metrika.co/dashboard/algorand/) - Algorand network performance and account monitor.
 - [Algorand Stats](https://www.algorandstats.com/) - All stats about Algorand Governance.
+- [ASA Stats](https://www.asastats.com/) - All Algorand assets on one dashboard
```
```suggestion
- [ASA Stats](https://www.asastats.com/) - All Algorand assets on one dashboard.
```

aorumbayev reviewed (changes requested):
@salvatorecorvaglia i wasn't talking about description of PR, the linter runs on the readme file - you are missing the `.` in description inside the readme. Please apply this change and we should be good to go

Commit:
Add ASA Stats on Metrics

ASA Stats aims to have accurate and up-to-date price information for every ASA and NFT that has ever been sold on Algorand.

salvatorecorvaglia commented:
ok, done.
